server/server.py

```diff
@@ -2,18 +2,22 @@
 from flask import Flask
 from slackeventsapi import SlackEventAdapter
 from command.core import Core
+import logging
 
 app = Flask(__name__)
 core = Core()
+logging.basicConfig(format='%(asctime)s - %(levelname)s : %(message)s',
```

Comment:
Would you add `%(module)s` and maybe `%(funcName)s` to the config? That would add the module and function doing the calling, which would be more convenient for debugging.